### ISSUE TITLE
question about test dataset on mvtec dataset

### ISSUE THREAD

**Issue by tangsankou** [pangdatangtt/UniNet#27]
Hi, thanks for sharing your code! I noticed that during training the evaluation is performed directly on the test_dataloader. Did I understand correctly that there is no separate validation set, and the test set is used for evaluation throughout training?
